src/test/java/org/folio/innreach/domain/service/impl/ContributionValidationServiceImplTest.java

```diff
@@ -285,10 +291,14 @@ void throwWhenMultipleStatisticalCodes() {
   void testEligibleInstance_noStatisticalCodes() {
     when(contributionConfigService.getCriteria(any())).thenReturn(CRITERIA);
     when(holdingsService.find(any())).thenReturn(Optional.empty());
+    when(folioLocationService.getLocationLibraryMappings()).thenReturn(Map.of(TEST_UUID, TEST_UUID));
```

Comment:
return different values for location and library ids

Reply:
done